### PR TITLE
Header fancify

### PR DESCRIPTION
**Pull request by Icety**
Makes the header fancy :D

### CODE REVIEW

**Comment by martijn9612** [comment]
Changes look good, but the changes are not responsive. This has to be fixed. The text is way too big on a mobile screen and makes the page unreadable.

File: app/elements/lancie-home-page/lancie-image-slider.html [hunk 1]
```diff
@@ -28,16 +28,67 @@
...
+        color: rgb(255, 255, 255);
+        line-height: 1.2em;
+      }
+
+      .content .date {
+        font-size: 72px;
+        text-transform: uppercase;
+        letter-spacing: 10px;
+        font-weight: 500;
+        margin-bottom: 32px;
```
Comment (martijn9612) [inline app/elements/lancie-home-page/lancie-image-slider.html]: This margin should be changed to 64px to have the same amount of space between each element in the image-slider.

Comment (Icety) [inline app/elements/lancie-home-page/lancie-image-slider.html]: It is all responsive now.

File: app/elements/lancie-home-page/lancie-home-page.html [hunk 2]
```diff
@@ -121,6 +121,8 @@
     <lancie-section type="primary full no-padding">
       <google-map latitude="51.9951895" longitude="4.3765975" zoom="14" disable-zoom fit-to-markers>
         <google-map-marker latitude="51.999020" longitude="4.371175"
+      <google-map latitude="51.999211" longitude="4.372039" zoom="16" disable-zoom fit-to-markers>
```
Comment (TimvdLippe) [inline app/elements/lancie-home-page/lancie-home-page.html]: I think this should be removed?

Comment (Icety) [inline app/elements/lancie-home-page/lancie-home-page.html]: Yeah, you're right

File: app/elements/lancie-home-page/lancie-image-slider.html [hunk 3]
```diff
@@ -28,45 +28,142 @@
...
-      <div class="overlay">
-        <iron-image preload fade sizing="contain"
-          position="left" src="https://areafiftylan.nl/images/logos/lancie/logo_without_date.png"></iron-image>
-      </div>
-
       <div class="slider">
+        <div class="overlay"></div>
+        <div class="content layout vertical">
+          <div class="logo-frontpage">
+            <!--<img class="logo-frontpage" src="https://areafiftylan.nl/images/logos/lancie/logo_without_date.png">-->
```
Comment (TimvdLippe) [inline app/elements/lancie-home-page/lancie-image-slider.html]: If this tag is not needed anymore, please remove it.

**Comment by TimvdLippe** [comment]
Looks good and responsive. 2 small comments then LGTM

**Comment by martijn9612** [comment]
There is still one issue with the image scroll being too wide. On mobile, it is possible to scroll the site to the right and look at a empty space. It has to do with the width of the scrolling images, the images are of correct size but the container is not.

**Pull request merged**